compiler/expr_visitor_test.py

```diff
@@ -35,11 +35,13 @@ def Test(self):
   return Test
 
 
-def _MakeLiteralTest(lit):
+def _MakeLiteralTest(lit, expected=None):
   def Test(self):
     status, output = _GrumpRun('print repr({!r}),'.format(lit))
```

Comment:
Hmm this is bit of a weird case because the result of repr() on a set does not give back the same literal string that produced it, so I see why you're changing this. But I don't think it's working quite the way we want. In this case Grumpy is running the statement `print repr(set(['foo', 'bar']))`. It would be better if it was running `print repr({'foo', 'bar'})`. Maybe the best thing to do is for lit to be a string value that we pass directly into _GrumpRun and then compare the output with `repr(ast.literal_eval(lit))`.

I think the effect is equivalent for the other kinds of literals that we test, but let me know if you think that's not the case.